src/extension.ts

```diff
@@ -119,6 +121,18 @@ export async function activate(context: vscode.ExtensionContext) {
         }
     );
 
+    const currVersionReleaseName =
+        "release_note_" + getPackageInfo(context).extensionVersion;
+    const viewedReleaseNote = context.globalState.get(
+        currVersionReleaseName,
+        false
+    );
+
+    if (!viewedReleaseNote) {
+        PopupService.openReleaseNote();
+        context.globalState.update(currVersionReleaseName, true);
+    }
```

Comment:
My understanding is that every time we update the extension, users who receive the update will be shown `LATEST_RELEASE_NOTE` .
When developers are ready for a new version, they need to NOT forget to update `LATEST_RELEASE_NOTE`. Otherwise on update VS Code will show the old release notes.
I think that adds overhead to updating the extension. What can we do to streamline the process?

Reply:
I still need to look into an HTML loader that can load an HTML file into a string. Then, it might be easier to just select the newest release note and make sure that certain attributes can match the actual version number before loading it in. This is just a quick solution, since we're on a tight deadline :((((
Otherwise, I can rename the variable in latest_release_note.ts to be more unique to the release date so that it's more noticeable which release note it's pointing to! 

What do you think?

Reply:
I think it's good to go for this release. Thanks for the clarification!